tests/System.IO.Pipelines.Tests/SocketsFacts.cs

```diff
@@ -173,6 +173,49 @@ public void CanCreateWorkingEchoServer_PipelineSocketServer_NonPipelineClient()
             }
         }
 
+        [Fact(Skip = "System.TypeLoadException : A value type containing a by-ref instance field, such as Span<T>, cannot be used as the type for a class instance field.")]
+        public async Task RunStressPingPongTest_RioServer_LibuvClient()
+        {
+            var endpoint = new IPEndPoint(IPAddress.Loopback, 5060);
+            const int SendCount = 500, ClientCount = 5;
+
+            var runServerTask = Task.Factory.StartNew(() =>
+            {
+                var endpointBytes = endpoint.Address.GetAddressBytes();
+
+                var server = new RioTcpServer((ushort)endpoint.Port, endpointBytes[0], endpointBytes[1], endpointBytes[2], endpointBytes[3]);
+
+                for (int loop = 0; loop < ClientCount; loop++)
+                {
+                    var connection = server.Accept();
+                    var task = PongServer(connection);
+                }
+
+                server.Stop();
```

Comment:
Is this a race with `PongServer`, which isn't awaited? I didn't get any issues running this, but it could still be sketchy...